azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/middleware/sdk/ChatActionHandler.kt

```diff
@@ -167,6 +168,9 @@ internal class ChatActionHandler(private val chatService: ChatService) {
     private fun initialization(dispatch: Dispatch, threadId: String) {
         try {
             chatService.initialize()
+            chatService.getAdminUserId()?.let {
+                dispatch.invoke(ParticipantAction.ParticipantToHideReceived(it))
+            }
```

Comment:
Could you point me where have you created a readReceiptMap which contains all the participants of the chat? Having that in another place in our code might cause a race condition.